deployment/ansible/roles/model-my-watershed.celery-worker/templates/nginx-flower.conf.j2

```diff
@@ -0,0 +1,37 @@
+server {
+  listen *:80;
+  server_name _;
+
+  access_log /var/log/nginx/flower.access.log logstash_json;
+
+  {% if ['packer'] | is_in(group_names) -%}
+  location /version.txt {
+    alias /srv/version.txt;
```

Comment:
Minor, but version.txt on app/tiler shows the long sha1 and version.txt shows the short sha1.